apps/backend/src/handlers/getAllForms.ts

```diff
@@ -6,27 +6,44 @@ import { fetchDocuments } from '../db/utils.js';
  * An HTTP get method to get all forms from the QLDB table.
  */
 export const getAllFormsHandler = async (event: APIGatewayEvent) => {
+  const headers = {
+    'Access-Control-Allow-Headers': 'Content-Type, Access-Control-Allow-Origin',
+    'Access-Control-Allow-Origin': '*',
+    'Access-Control-Allow-Methods': 'GET',
+  };
+
   if (event.httpMethod !== 'GET') {
     return {
       statusCode: 400,
-      body: `getMethod only accepts GET method, you tried: ${event.httpMethod} method.`
-    }
+      headers,
+      body: `getMethod only accepts GET method, you tried: ${event.httpMethod} method.`,
+    };
   }
   // All log statements are written to CloudWatch
   console.info('received:', event);
 
-  await createTableIfNotExists();
-  const allForms = await fetchDocuments();
+  try {
+    await createTableIfNotExists();
+    const allForms = await fetchDocuments();
 
-  const response = {
-    statusCode: 200,
-    body: JSON.stringify(allForms)
-  };
+    const response = {
+      statusCode: 200,
+      headers,
+      body: JSON.stringify(allForms),
+    };
 
-  // All log statements are written to CloudWatch
-  console.info(
-    `response from: ${event.path} statusCode: ${response.statusCode}`
-  );
+    // All log statements are written to CloudWatch
+    console.info(
+      `response from: ${event.path} statusCode: ${response.statusCode}`
+    );
 
-  return response;
-};
\ No newline at end of file
+    return response;
+  } catch (error) {
+    console.log('error accessing database');
```

Comment:
can you make this `console.error(error)`? So we get the whole error log in Cloudwatch if something goes wrong

Reply:
also were you able to test this full-stack?